<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SudheerMyra" ignored="0" name="Test" tests="5" failures="5" timestamp="2023-08-14T16:01:00 CDT" time="65.09" errors="0">
  <testcase name="test_LoginDDT" time="10.993" classname="testCases.TC_003_LoginDataDrivenTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.test_LoginDDT(TC_003_LoginDataDrivenTest.java:52)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_LoginDDT -->
  <testcase name="test_LoginDDT" time="10.681" classname="testCases.TC_003_LoginDataDrivenTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.test_LoginDDT(TC_003_LoginDataDrivenTest.java:52)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_LoginDDT -->
  <testcase name="test_LoginDDT" time="10.737" classname="testCases.TC_003_LoginDataDrivenTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.test_LoginDDT(TC_003_LoginDataDrivenTest.java:52)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_LoginDDT -->
  <testcase name="test_LoginDDT" time="10.724" classname="testCases.TC_003_LoginDataDrivenTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.test_LoginDDT(TC_003_LoginDataDrivenTest.java:52)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_LoginDDT -->
  <testcase name="test_LoginDDT" time="10.784" classname="testCases.TC_003_LoginDataDrivenTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.test_LoginDDT(TC_003_LoginDataDrivenTest.java:52)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_LoginDDT -->
</testsuite> <!-- Test -->
